devices/YarpRobotLoggerDevice/README.md

```diff
@@ -1,47 +1,38 @@
-### iCubGenova04 strain2 FT-IMU logger
+# YARPRobotLoggerDevice
 
-A logger device based on `YarpSensorBridge` and `matioCpp` to record dataset from strain2 sensor boards mounted on the iCubGenova04 robot.
+The **YARPRobotLoggerDevice** is a YARP device based on `YarpSensorBridge` and `matioCpp` to record dataset from a YARP-based robot.
 
 
 
-### Launching
+## :running: How to use the device
 
-- Launch yarprobotinterface on the robot with `icub_wbd_inertials.xml`
+- Launch yarprobotinterface on the robot 
 
+- Launch the logger device after properly setting the `YARP_ROBOT_NAME` environment variable
   ```
-  yarprobotinterface --config icub_wbd_inertials.xml
-  ```
-
-- Launch the FT-IMU logger device,
-  ```
-  YARP_ROBOT_NAME=iCubGenova04 yarprobotinterface --config launch-ft-imu-logger.xml
+  yarprobotinterface --config launch-yarp-robot-logger.xml
   ```
 
 - Press Ctrl+c to close the device, and the dataset is stored as the device is closed.
 
- Each dataset contains,
-
- - a `time` vector with the receive time stamps.
- - a struct for each strain-2 sensor attached on the robot (`l_foot_ft_imu`, `r_foot_ft_imu`, `r_leg_ft_imu`, `l_leg_ft_imu`) with each struct containing the corresponding FT sensor measurement, accelerometer, gyroscope and orientation sensor measurements.
-
-Associated sensors,
-- `l_leg_ft_imu`
-    - FT: `l_leg_ft_sensor`
-    - acc: `l_upper_leg_ft_acc_3b12`
-    - gyro: `l_upper_leg_ft_gyro_3b12`
-    - orientation: `l_upper_leg_ft_eul_3b12`
-- `l_foot_ft_imu`
-    - FT: `l_foot_ft_sensor`
-    - acc: `l_foot_ft_acc_3b13`
-    - gyro: `l_foot_ft_gyro_3b13`
-    - orientation: `l_foot_ft_eul_3b13`
-- `r_leg_ft_imu`
-    - FT: `r_leg_ft_sensor`
-    - acc: `r_upper_leg_ft_acc_3b11`
-    - gyro: `r_upper_leg_ft_gyro_3b11`
-    - orientation: `r_upper_leg_ft_eul_3b11`
-- `r_foot_ft_imu`
-    - FT: `r_foot_ft_sensor`
-    - acc: `r_foot_ft_acc_3b14`
-    - gyro: `r_foot_ft_gyro_3b14`
-    - orientation: `r_foot_ft_eul_3b14`
+## Data
+
+ Depending on the configuration file, each dataset can contain:
+ - `time` vector with the receive time stamps.
```

Comment:
This part is a bit tricky each element of the `time` vector contains the time of the last read done by the sensor bridge 

https://github.com/ami-iit/bipedal-locomotion-framework/blob/f017cea7786048a8239f6434f6e58c7782906326/devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp#L328

I think we should update the code, possible solutions are:
1. Have a time vector for each data collected by the sensorbridge 😢 
2. Keep only one timestamp and change this line
   
   https://github.com/ami-iit/bipedal-locomotion-framework/blob/f017cea7786048a8239f6434f6e58c7782906326/devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp#L381

  with 
  ```cpp
   m_time.row(bufferSize) << yarp::os::Time::now()
  ```
  In this case, the data should be synchronized among different process that logs them 

cc @isorrentino and @traversaro

Reply:
By the way, this PR is not blocked by this comment. 😃

Reply:
@GiulioRomualdi actually I was testing the second solution you proposed right now. See https://github.com/isorrentino/bipedal-locomotion-framework/commit/a892082ae164996480f91d828c43b5185003439a.

Reply:
> I was testing the second solution you proposed right now. 

I opened a PR https://github.com/ami-iit/bipedal-locomotion-framework/pull/473

Reply:
Since the PR has been merged, I have updated this line